tests20/python_client/testcases/test_partition.py

```diff
@@ -442,7 +445,7 @@ def test_partition_release_dropped_collection(self):
         expected: raise exception
         """
         m_collection = self._collection()
-        p_name = gen_unique_str(prefix)
+        p_name = cf.gen_unique_str(prefix)
         m_partition, _ = self.partition.partition_init(m_collection, p_name)
         assert m_collection.has_partition(p_name)
```

Comment:
self.collection.has_partition(p_name) is better later

Reply:
ok